docs/Operators.md

```diff
@@ -5144,6 +5144,15 @@ This version of the operator has been available since version 1 of the default O
 ### <a name="LpNormalization"></a><a name="lpnormalization">**LpNormalization**</a>
 
   Given a matrix, apply Lp-normalization along the provided axis.
+  
+  For p=2,
+  
+  y = x  / (sqrt(sum(x ** 2)) + epsilon),
```

Comment:
This might make it difficult for converter, due to how each framework add epsilon:

    TF: x / sqrt(max(sum(x**2), epsilon))
    Others: x / sqrt(sum(x**2) + epsilon)

Reply:
Any suggestion?

Reply:
I don't have a good solution. What Caffe2 and PyTorch use?

Reply:
caffe2 doesn't have epsilon, pytorch uses `max(sqrt(sum(x**2)), epsilon)`
yep yet another two variants :)

Reply:
so what is the resolution for this?

Reply:
@bddppq @ebarsoum ping?

Reply:
If both TF and PT use max, and based on both equations we can easily convert between them. Then let's use max and not plus.

Reply:
Also, max doesn't add bias like the plus.